### DISCUSSION TITLE
Person names

### DISCUSSION THREAD

**Discussion by neilotte** [CommonCoreOntology/CommonCoreOntologies#391] [Ontology Terms and Class Schema]
Request for First Name, Middle Name, Last Name for persons.

**Comment by APCox** [comment]
These terms already exist, but are currently in an extension ontology. They are being reviewed for promotion to the CCO.

**Comment by neilotte** [comment]
@APCox (Has it really been five years? How time flies...) 

Any thoughts on this migration?

**Comment by swartik** [comment]
I have no knowledge of the extension ontology, so this comment may be misinformed. But having First Name, Middle Name, and Last Name seems too culture-specific for CCO. In some Asian cultures the surname comes first. Many persons have multiple middle names. A mid-level ontology ought to accommodate name parts the world over.

**Comment by alanruttenberg** [comment]
Some people have only a single name, e.g. [Prince](https://en.wikipedia.org/wiki/Prince_(musician))

**Comment by neilotte** [comment]
@alanruttenberg @APCox @swartik Are any of you aware of an existing standards--say an ISO standard--for names? We should be re-using here.

**Comment by alanruttenberg** [comment]
I do not. I could look but this might be something @jimschoening knows about since it falls in the realm of the personal information ontology stuff he is doing.

**Comment by alanruttenberg** [comment]
There's this, which is useful to be aware of: [Falsehoods Programmers Believe About Names](https://shinesolutions.com/2018/01/08/falsehoods-programmers-believe-about-names-with-examples/)

**Comment by alanruttenberg** [comment]
And this: [Personal names around the world](https://www.w3.org/International/questions/qa-personal-names) from the W3C

**Comment by swartik** [comment]
Considering the two links @alanruttenberg provides – and let's not even get into the period when Prince was referred to as The Artist Formerly Known as Prince – a standard for names seems unlikely to be universally applicable. I do not mean to say CCO shouldn't support name components. There should, however, be no expectation that a person has a given name and a surname, or that a given name is a first name.

I can envision CCO including classes GivenName and Surname, if their definitions clearly state their limitations. Including them would acknowledge CCO's U.S. origins and expected early user community. Eventually these classes might be deprecated in favor of classes in culture-specific extension ontologies.

**Comment by neilotte** [comment]
Moving to discussion.
